### ISSUE TITLE
Help Me

### ISSUE THREAD

**Issue by utpal4job** [numerai/numerapi#106]
Hello, i am new in numerai and i have checked your example.  on the example you have csv file called "preds.csv", how can i generate "preds.csv" file?

Thanks,

**Comment by Nimmerfall** [comment]
would be something like (i assume you already have most of the steps):
import pandas as pd
napi = NumerAPI(public_id="your public api key", secret_key="yor secret api key")

'# you don't have to, but in this example it's used to identify the current prediction
current_round = napi.get_current_round()

'# download live data and save it somwhere
napi.download_dataset("v4.2/live_int8.parquet", f"live_int8.parquet")
'# load the parquet
live_data = pd.read_parquet(f"live_int8.parquet")

'# make the predictions -> fillna only for safety
live_data['prediction'] = your_model.predict(live_data[features_to_use])

'# then write predictions to csv file
live_data["prediction"].to_csv("prediction{current_round}.csv")

model_id = napi.get_models()['your_model'] 
napi.upload_predictions("prediction{current_round}.csv", model_id=model_id)

So basically you just add your prediction as column to the live dataframe and write this column to csv

**Comment by utpal4job** [comment]
Hello, thanks for your reply. I will try and let you know.

Thanks.

**Comment by utpal4job** [comment]
Hello, one thing i am not getting, how can i add RSI buy/sell conditions on prediction column. 

Pls help me to make the RSI buy/sell conditions with specific value.

Thanks.
